docs/frontend_utilisation_guide.md

```diff
@@ -76,15 +120,21 @@ document.body.onload = function () {
 ```typescript
 const config = new Config({ useUrlParams: true, initialSettings: { OfferToReceive: true, TimeoutIfIdle: true } });
 ```
-6) Create the `stream` object and pass in the new `config` object
+6) Create an if statement that will make use of the `webSocketAddress` variable if one is included 
```

Comment:
where is step 5?

Reply:
Right above step 6 ;) This only shows the change not the whole file you can click the two facing arrows and it shows step 5.